yapftests/yapf_test.py

```diff
@@ -505,6 +505,16 @@ def testInPlaceReformattingEmpty(self):
         reformatted_code = fd.read()
     self.assertEqual(reformatted_code, expected_formatted_code)
 
+  def testPrintModified(self):
+    for unformatted_code, has_change in [('1==2', True), ('1 == 2', False)]:
+      with utils.TempFileContents(
+          self.test_tmpdir, unformatted_code, suffix='.py') as filepath:
+        output = subprocess.check_output(
+            YAPF_BINARY + ['--in-place', '--print-modified', filepath],
+            text=True)
+        check = self.assertIn if has_change else self.assertNotIn
+        check('Formatted %s' % filepath, output)
```

Comment:
Given that Python support for Yapf is at Python >=3.7 by now, how about:
```diff
--- check('Formatted %s' % filepath, output)
+++ check(f'Formatted {filepath}', output)
```
One more cases like that further up. Just an idea.

Reply:
Done.